v22.1/demo-fault-tolerance-and-recovery.md

```diff
@@ -197,27 +191,27 @@ Initially, the workload creates a new database called `ycsb`, creates a `usertab
 
 1. Go to the DB Console at <a href="http://localhost:8080" data-proofer-ignore>http://localhost:8080</a>.
 
-2. To check the SQL queries getting executed, click **Metrics** on the left, and hover over the **SQL Queries** graph at the top:
+2. To check the SQL queries getting executed, click **Metrics** on the left, and hover over the **SQL Statements** graph at the top:
 
-    <img src="{{ 'images/v22.1/fault-tolerance-1.png' | relative_url }}" alt="DB Console" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/fault-tolerance-1.png' | relative_url }}" alt="DB Console SQL Statements" style="border:1px solid #eee;max-width:100%" />
 
-3. To check the client connections from the load generator, select the **SQL** dashboard and hover over the **SQL Connections** graph:
+3. To check the client connections from the load generator, select the **SQL** dashboard and hover over the **Open SQL Sessions** graph:
 
-    <img src="{{ 'images/v22.1/fault-tolerance-2.png' | relative_url }}" alt="DB Console" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/fault-tolerance-2.png' | relative_url }}" alt="DB Console Open Sessions" style="border:1px solid #eee;max-width:100%" />
 
     You'll notice 3 client connections from the load generator. If you want to check that HAProxy balanced each connection to a different node, you can change the **Graph** dropdown from **Cluster** to specific nodes.
 
-4. To see more details about the `ycsb` database and `usertable` table, click **Databases** in the upper left and then scroll down until you see **ycsb**:
+4. To see more details about the `ycsb` database and `usertable` table, click **Databases** in the upper left and click **ycsb**:
```

Comment:
Based on the new screenshots, does the table need to be called `public.username` throughout this doc? Not sure of the technicalities.